packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/fpsrange/FpsRangeFeature.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.fpsrange;
+
+import android.hardware.camera2.CaptureRequest;
+import android.util.Range;
+import io.flutter.plugins.camera.CameraProperties;
+import io.flutter.plugins.camera.features.CameraFeature;
+
+/**
+ * Controls the frames per seconds (FPS) range configuration on the {@link android.hardware.camera2}
+ * API.
+ */
+public class FpsRangeFeature extends CameraFeature<Range<Integer>> {
+  private Range<Integer> currentSetting;
+
+  /**
+   * Creates a new instance of the {@link FpsRangeFeature}.
+   *
+   * @param cameraProperties Collection of characteristics for the current camera device.
+   */
+  public FpsRangeFeature(CameraProperties cameraProperties) {
+    super(cameraProperties);
+
+    Range<Integer>[] ranges = cameraProperties.getControlAutoExposureAvailableTargetFpsRanges();
+
+    if (ranges != null) {
+      for (Range<Integer> range : ranges) {
+        int upper = range.getUpper();
+
+        // There is a bug in the Pixel 4A where it cannot support 60fps modes
```

Comment:
Is there any reasonable way we could gate this workaround to that device, rather than changing the behavior for all Android devices?

Reply:
Yes it is possible, I updated the code accordingly